shared/middleware/helpers/property_bag.rb

```diff
@@ -114,6 +116,12 @@ def set(name, value, ip_address, time = DateTime.now)
       },
     )
     value
+  rescue Aws::DynamoDB::Errors::ValidationException => e
+    if e.message == 'Item size has exceeded the maximum allowed size'
+      too_large
```

Comment:
you could just raise TooLarge here? Who is handling these exceptions and turning them into http statuses?